imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -147,6 +216,11 @@ Template.searchModal.helpers({
   },
   showSearchResults() {
     return false;
+  },
+  hasResults() {
+    const instance = Template.instance();
+    const sortResults = instance.state.get("productSearchResults").length;
+    return sortResults > 0;
```

Comment:
remove this return and above the line use `return instance.state.get("productSearchResults").length; > 0`